src/main/java/org/broadinstitute/consent/http/resources/DataRequestVoteResource.java

```diff
@@ -164,8 +164,8 @@ public Response describeDataOwnerVote(@PathParam("requestId") String requestId,
     @GET
     @Produces("application/json")
     @PermitAll
-    public List<Vote> describeAllVotes(@PathParam("requestId") String requestId) {
-        return voteService.describeVotes(requestId);
+    public List<Vote> describeVotesOnElections(@PathParam("requestId") String requestId, List<Integer> electionIds) {
+        return voteService.findVotesByElectionIds(electionIds);
     }
```

Comment:
This should be a new API, not a modification of an existing one. We need to maintain backwards compatibility and support outdated/deprecated APIs for at least one release cycle before removing them.